securedrop/config.py.example

```diff
@@ -83,3 +83,6 @@ TEMP_DIR = os.path.join(SECUREDROP_DATA_ROOT, "tmp")
 # depending on the environment
 DATABASE_ENGINE = 'sqlite'
 DATABASE_FILE = os.path.join(SECUREDROP_DATA_ROOT, 'db.sqlite')
+
+# Which of the available locales should be displayed by default ?
+DEFAULT_LOCALE = 'en_US'
```

Comment:
Thinking ahead, this value should probably be configurable. There should be a default role var in the "app" role, e.g. `securedrop_default_locale`, and be set to "en_US".